epics-team-builder/src/widgets/SetupPage/components/ManuallyAssignProjects.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { Card, Table, Button, CardDeck } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+export default class ManuallyAssignProjects extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      projectData: [{}],
+      studentData: [{}]
+    };
+  }
+
```

Comment:
We can just delete the constructor all together since we aren't using state in this component